docs-source/content/faq/node-heating.md

```diff
@@ -0,0 +1,56 @@
+
+To use these features, edit the Domain Custom Resource to add content to the `serverPod` element, in this case at the scope of a cluster, as shown in the following example:
+
+```
+clusters:
+- clusterName: cluster-1
+  serverStartState: "RUNNING"
+  serverPod:
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                  - key: "weblogic.clusterName"
+                    operator: In
+                    values:
+                    - $(CLUSTER_NAME)
+              topologyKey: "kubernetes.io/hostname"
+```
+
+Because the `serverPod` element here is scoped to a cluster, the content of the `affinity` element will be added to the Pod generated for each WebLogic Server instance that is a member of this WebLogic cluster. This inter-pod anti-affinity statement expresses a preference that the scheduler select a Node for the new Pod avoiding as much as possible Nodes that already have Pods with the label "weblogic.clusterName" and the name of this cluster. Note that the `weight` is set to `100`, which is the maximum weight, so that this term will outweigh any possible preference for a Node based on availability of Docker images.
+
+It is possible to express many other scheduling preferences or constraints. The following example similarly expresses an anti-affinity, but changes the test to have all WebLogic Server instances in the domain prefer to run on Nodes where there is not already a Pod for a running instance:
+
+```
+serverPod:
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: "weblogic.domainUID"
+              operator: In
+              values:
+              - $(DOMAIN_UID)
+          topologyKey: "kubernetes.io/hostname"
+```
+
+Details about how the WebLogic Operator generates Pods for WebLogic Server instances, including details about labels and variable substitution, are available [here]({{< relref "/userguide/managing-domains/domain-resource#pod-generation" >}}).
+
```

Comment:
It could be useful to to have an example that covers all WL pods - not just for a particular domain-uid or cluster.   Eg. Any pod with a weblogic.domainUID - not just a particular domainUID.

Also, what is the precedence hierarchy for anti-affinity? In other words, what happens when multiple anti-affinity expressions are in effect?  E.g. any && domain && cluster

Reply:
Affinity content from the various scopes (domain, admin, cluster, etc.) are merged, so you get content with all of the terms from any of the scopes that apply.  So, yes, you can do use cases where cluster members have anti-affinity with each other and where all servers in the domain have some other kind of affinity or anti-affinity with other workloads.

Reply:
I'm not sure that I want more examples here unless we have a use case customers have described.

Reply:
So why not have the samples and FAQ emphasize 'all'?